lib/coverband/integrations/resque.rb

```diff
@@ -4,7 +4,7 @@
   Coverband.start
   Coverband.runtime_coverage!
   # no reason to miss coverage on a first resque job
-  Coverband::Collectors::Delta.set_default_results
+  Coverband.coverage.disable_skip_next_report!
```

Comment:
@danmayer - what do you think of something like this?

Reply:
nice much more understandable, big +1